webknossos-datastore/app/com/scalableminds/webknossos/datastore/controllers/BinaryDataController.scala

```diff
@@ -176,9 +179,10 @@ class BinaryDataController @Inject()(
             DataServiceRequestSettings(halfByte = halfByte))
           imageProvider <- respondWithSpriteSheet(dataSource, dataLayer, request, 1, blackAndWhite)
         } yield {
-          Ok.stream(Enumerator.outputStream(imageProvider).andThen(Enumerator.eof)).withHeaders(
-            CONTENT_TYPE -> contentTypeJpeg,
-            CONTENT_DISPOSITION -> "filename=test.jpg")
+          Result(
+            header = ResponseHeader(200),
+            body = HttpEntity.Streamed(StreamConverters.asOutputStream().mapMaterializedValue { outputStream => imageProvider(outputStream) }, None, Some(contentTypeJpeg)))
+          //TODO: disposition
```

Comment:
Not sure if we even need the content-disposition, but then we can remove the TODO